std/random.d

```diff
@@ -1402,6 +1402,17 @@ if (!is(CommonType!(T1, T2) == void))
     auto b = uniform(0.0f, 1.0f, gen);
 }
 
+/// Create an array of random numbers using range functions and UFCS
+@safe unittest
+{
+    import std.array : array;
+    import std.algorithm.iteration : iota, map;
+
+    int[] arr = iota(100).map!(() => uniform(0, 100)).array;
+    assert(arr.length == 100);
+    assert(arr[0] >= 0 && arr[0] < 100);
+}
```

Comment:
Ideas:
- `arr.sum > 0`
- use a seed
- seed the global RNG before

I would go with (b). It's common practice to use explicit seeding with RNGs.